src/utils/profileFormHelpers.ts

```
@@ -0,0 +1,104 @@
+/**
+ * Helper functions for ProfileForm operations
+ * Handles data preparation, localStorage updates, and business logic
+ */
+
+import { NotificationToast } from 'components/NotificationToast/NotificationToast';
+import { urlToFile } from 'utils/urlToFile';
+import { removeEmptyFields } from 'utils/userUpdateUtils';
+import { errorHandler } from 'utils/errorHandler';
+
+export const prepareUpdatePayload = async (
+  formState: any,
```

ESLint

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] Insert `,`

(prettier/prettier)